Assets/WizardsCode/Character/Scripts/Ink/TextBubbleComp.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using TMPro;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+// todo -find a permanent home for this, it wouldn't let it be in my own directory
```

Comment:
Its your repo, I wasn't sure where to put this file after unity failed to find it in my _nathan directory. Where would you like it?